tests/integration/test_ceph.py

```diff
@@ -0,0 +1,186 @@
+import time
+import unittest
+import subprocess
+from ims.common import config
+
+config.load()
+
+import rados
+import rbd
+from ims.common import shell
+from ims.common import constants
+from ims.einstein import ceph
+from ims.einstein.operations import BMI
+from ims.exception import file_system_exceptions
+from ims.common.log import trace
+import logging
+
+_cfg = config.get()
+
+POOL = _cfg.fs.pool
+CEPH_IMG = "BMI_TEST_CEPH"
+TEST_NOT_EXIST_IMG_NAME = "BMI_TEST_IMG_NOT_FOUND"
+CEPH_IMG_SIZE = 1024
+INDEX_OUTSIDE_THE_IMAGE = CEPH_IMG_SIZE + 1
+CEPH_SNAP_IMG = "BMI_TEST_CEPH_SNAP"
+TEST_DATA = 'bmi' * 10
+OFFSET = 0
+CEPH_CHILD_IMG = "BMI_TEST_CEPH_CHILD_IMAGE"
+
+
+class TestCreateImage(unittest.TestCase):
```

Comment:
> This is because setUp and tearDowns are different for each class. I will advice against grouping tests together if they don't belong together even though they have same setUps and tearDowns.

I would agree with you if this is unit test, for integration tests we want to check functionality.